scripts/settings.lua

```diff
@@ -199,6 +199,33 @@ BCSettingsBySubtitle = {
 			},
 		},
 	},
+	{
+	title = "bc_missionGeneration",
+	elements = {
+		{name = "generationInterval",
+		values = {1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24},
+		texts = {"1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24"},
+		default = 1,
+		title = "bc_generationInterval",
+		tooltip = "bc_generationInterval_tooltip",
+		actionFunc = function(self,ix)
+			BetterContracts:updateGenerationSettings() -- recalculate generation settings
+			end,
+		noTranslate = true
+			},
+		{name = "missionGenPercentage",
+		values = {0.01, 0.02, 0.04, 0.05, 0.1, 0.2},
+		texts = {"1%", "2%", "4%", "5%", "10%", "20%"},
```

Comment:
I took only values 100 can be divided by. Feel free to suggest changes

Reply:
Hm,
although I can understand your logic, it feels a little too-much for me. Mission generation is a somewhat random process: they look at random fields, and try to find an appropriate field work type. It could happen that MAX_TRIES_PER_GENERATION is reached before a single mission was created. 
But we should give it a try and see if it helps distributing  mission generation more evenly.

Reply:
Yeah I don't fully understand what MAX_TRIES_PER_GENERATION  does, but without being able to adjust MAX_MISSIONS_PER_GENERATION, you would always have the maximum amount of missions after 5 generations in my tests. 
I tried setting 1h and 1% on Somewhere in Thuringia Map (about 160 fields) and got about 1 contract every hour (with MAX_MISSIONS being limited to 80) so it seems to work.